src/site/content/en/blog/value-of-speed/index.md

```diff
@@ -0,0 +1,200 @@
+    drop-down list, and click **Create**.
+
+    * Once the report has been generated, click **Select a metric >
+    Metric Group >** **Ecommerce Conversion Rate**.
+    ![image](conversion-rate.jpg)
+    <br><br>
+
+    * Select the **Week** view for the dataset.
+    ![image](landing-pages-week.jpg)
+    <br><br>
+
+    * Select a time period that covers 2–3 months before the speed
+    optimization and 2–3 months after. (In our example, that's January 2018
+    through October 2018.)
+    ![image](analytics-start-date.jpg)
+    <br><br>
+
+    * Export the report to [Google Sheets](http://sheets.google.com) or
+    your preferred spreadsheet application.
+    ![image](export-google-sheets.jpg)
+    <br><br>
+
+1. Now get the speed and conversion data for the desktop site. Open
+[this custom report](https://analytics.google.com/analytics/web/template?uid=X21Nb_soQp69U_ylfNmLvg)
+in Google Analytics and repeat the process for step 1 to export the data to
+a spreadsheet.
+
+1.  Copy the _Ecommerce Conversion Rate_ column in the desktop report
+spreadsheet and insert it next to the _Ecommerce Conversion Rate_ in the
+mobile report spreadsheet.
+
+1.  To calculate the Rel mCvR, create a new column titled _Relative
+Mobile Conversion Rate (Rel mCvR)_. Then add a formula to divide the mobile
+conversion rate by the desktop conversion rate.
+<br><br>
+![image](relative-rate-formula.jpg)
+<br><br>
+
+1. Create a chart based on the _Mobile Avg. Page Load Time (sec)_ and
+_Relative Mobile Conversion Rate (Rel mCvR)_ columns. To do that in Google
+Sheets:
+
+    * Click **Insert > Chart**.
+    * In the **Chart Editor** pane, select the **Line chart** chart type.
+    * Select the _Week Index_, _Mobile Avg. Page Load Time (sec)_, and
+      _Relative Mobile Conversion Rate (Rel mCvR)_ columns as the data ranges.
+    * Click the **Customize** tab and then select **Series**.
+    * In the **Series selector** drop-down list, select **Relative
+      Mobile Conversion Rate (Rel mCvR)**.
+    * In the **Axis** drop-down list select **Right axis**.
+    * If you like, click **Chart & axis titles** to customize the chart title
+      and the left and right _y_-axis labels.
+
+  You should now have a chart that looks something like this:
+  ![image](mobile-versus-relative-chart.jpg)
+  <br><br>
+
+1. Using the chart, identify a period before the speed optimization
+(when load times were high) and a period after the speed optimization (when
+load times should be lower) that you want to analyze. In this example, you
+would compare eight weeks in Jan–Feb to eight weeks in Aug–Sept.
+
+1. In a new sheet, calculate the average load time and rel mCvR for the
+two periods. Then add the revenue coming from mobile visitors during the
+period after the speed optimization (Aug–Sept in the example). You find
+revenue data in Google Analytics under the section **Audience > Mobile >
+Overview**.
+
+  ![image](period-revenue.jpg)
```

Comment:
I'd suggest giving these images a max-width and centering them so they fit a bit better with the surrounding text.

```
<figure class="w-figure  w-figure--center">
  <img src="./period-revenue.jpg" alt="Revenue data for January through September showing Rel mCvR" style="max-width: 600px; width: 100%">
  <figcaption class="w-figcaption">
    An optional caption if you want to add one. Otherwise you can delete the figcaption element.
  </figcaption>
</figure>
```

Reply:
Good idea.

I've changed them all.